src/VideoIntelligence/V1beta1/VideoIntelligenceServiceClient.php

```diff
@@ -0,0 +1,428 @@
+ *
+ * EXPERIMENTAL: this client library class has not yet been declared beta. This class may change
+ * more frequently than those which have been declared beta or 1.0, including changes which break
+ * backwards compatibility.
+ */
+
+namespace Google\Cloud\VideoIntelligence\V1beta1;
+
+use Google\GAX\AgentHeaderDescriptor;
+use Google\GAX\ApiCallable;
+use Google\GAX\CallSettings;
+use Google\GAX\GrpcConstants;
+use Google\GAX\GrpcCredentialsHelper;
+use Google\GAX\LongRunning\OperationsClient;
+use Google\GAX\OperationResponse;
+use google\cloud\videointelligence\v1beta1\AnnotateVideoProgress;
+use google\cloud\videointelligence\v1beta1\AnnotateVideoRequest;
+use google\cloud\videointelligence\v1beta1\AnnotateVideoResponse;
+use google\cloud\videointelligence\v1beta1\Feature;
+use google\cloud\videointelligence\v1beta1\VideoContext;
+use google\cloud\videointelligence\v1beta1\VideoIntelligenceServiceGrpcClient;
+
+/**
+ * Service Description: Service that implements Google Cloud Video Intelligence API.
+ *
+ * EXPERIMENTAL: this client library class has not yet been declared beta. This class may change
+ * more frequently than those which have been declared beta or 1.0, including changes which break
+ * backwards compatibility.
+ *
+ * This class provides the ability to make remote calls to the backing service through method
+ * calls that map to API methods. Sample code to get started:
+ *
+ * ```
+ * try {
+ *     $videoIntelligenceServiceClient = new VideoIntelligenceServiceClient();
+ *     $inputUri = "";
+ *     $features = [];
+ *     $operationResponse = $videoIntelligenceServiceClient->annotateVideo($inputUri, $features);
+ *     $operationResponse->pollUntilComplete();
+ *     if ($operationResponse->operationSucceeded()) {
+ *       $result = $operationResponse->getResult();
+ *       // doSomethingWith($result)
+ *     } else {
+ *       $error = $operationResponse->getError();
+ *       // handleError($error)
+ *     }
+ *
+ *     // OR start the operation, keep the operation name, and resume later
+ *     $operationResponse = $videoIntelligenceServiceClient->annotateVideo($inputUri, $features);
+ *     $operationName = $operationResponse->getName();
+ *     // ... do other work
+ *     $newOperationResponse = $videoIntelligenceServiceClient->resumeOperation($operationName, 'annotateVideo');
+ *     while (!$newOperationResponse->isDone()) {
+ *         // ... do other work
+ *         $newOperationResponse->reload();
+ *     }
+ *     if ($newOperationResponse->operationSucceeded()) {
+ *       $result = $newOperationResponse->getResult();
+ *       // doSomethingWith($result)
+ *     } else {
+ *       $error = $newOperationResponse->getError();
+ *       // handleError($error)
+ *     }
+ * } finally {
+ *     $videoIntelligenceServiceClient->close();
+ * }
+ * ```
+ *
+ * Many parameters require resource names to be formatted in a particular way. To assist
```

Comment:
I have filed an issue for this here: https://github.com/googleapis/toolkit/issues/1276